src/main.rs

```diff
@@ -13,42 +13,47 @@ use crate::{
     validate::{account::AccountValidator, network::NetworkValidator},
 };
 use anyhow::{Context, Result};
-use clap::AppSettings;
+use clap::Parser;
 use concordium_rust_sdk::endpoints::Client;
 use env_logger::{Builder, Env};
 use rosetta::models::NetworkIdentifier;
-use structopt::StructOpt;
 
-#[derive(StructOpt)]
-struct App {
-    #[structopt(
+#[derive(Parser, Debug)]
+#[clap(
+    author = "Concordium Foundation",
+    about = "A server implementing the Rosetta API for the Concordium blockchain."
+)]
+struct Args {
+    #[clap(
         long = "network",
+        env = "NETWORK",
```

Comment:
I think in general it is better to not use such generic environment variables as `NETWORK` and `PORT`.

In other projects we have we follow the convention to have environment variables `BINARY_NAME_...` or something close to `BINARY_NAME`.

Reply:
Done in https://github.com/Concordium/concordium-rosetta/pull/7.